admin-dev/themes/default/template/controllers/orders/helpers/view/view.tpl

```diff
@@ -338,7 +338,7 @@
 										{foreach from=$order->getReturn() item=line}
 										<tr>
 											<td>{$line.date_add}</td>
-											<td>{$line.type}</td>
+											<td>{l s=$line.type}</td>
```

Comment:
Ok but maybe it's more judicious to translate here: https://github.com/PrestaShop/PrestaShop/blob/develop/classes/order/OrderReturn.php#L158
isn't ?